config/jobs/kubernetes-sigs/secrets-store-csi-driver/secrets-store-csi-driver-presets.yaml

```diff
@@ -0,0 +1,39 @@
+presets:
+- labels:
+    preset-azure-secrets-store-creds: "true"
+  env:
+  - name: AZURE_CLIENT_ID
+    valueFrom:
+      secretKeyRef:
+        name: azure-secrets-store-cred
+        key: clientid
+  - name: AZURE_CLIENT_SECRET
+    valueFrom:
+      secretKeyRef:
+        name: azure-secrets-store-cred
+        key: clientsecret
+  - name: RESOURCE_GROUP
+    valueFrom:
+      secretKeyRef:
+        name: azure-secrets-store-cred
+        key: resourcegroup
+  - name: SUBSCRIPTION_ID
+    valueFrom:
+      secretKeyRef:
+        name: azure-secrets-store-cred
+        key: subscriptionid
+  - name: TENANT_ID
+    valueFrom:
+      secretKeyRef:
+        name: azure-secrets-store-cred
+        key: tenantid
+  - name: KEYVAULT_NAME
+    valueFrom:
+      secretKeyRef:
+        name: azure-secrets-store-cred
+        key: keyvaultname
+  - name: KEY_VERSION
```

Comment:
Assuming this is to retrieve the secret object from keyvault, are we missing the object name here?

Reply:
The secret and key name are defaulted in here - https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/master/test/bats/azure.bats#L12-L13. Do you think we should configure it here and read it?